app/api/routers/breeze_buddy/merchant_entities/__init__.py

```
@@ -0,0 +1,195 @@
+    get_all_merchant_entities_handler,
+    get_merchant_entity_by_merchant_id_handler,
+    update_merchant_entity_handler,
+)
+
+router = APIRouter()
+
+
+@router.post("/merchant", response_model=MerchantEntityResponse, status_code=201)
+async def create_merchant_entity(
+    merchant_data: MerchantEntityCreate,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)